src/app/lib/utilities/episodeAvailability/index.js

```diff
@@ -28,9 +28,10 @@ const getEpisodeAvailability = pageData => {
     'availableFrom',
   ]);
 
+  const versions = path(['content', 'blocks', '0', 'versions'], pageData) || [];
```

Comment:
```suggestion
  const episodeHasExpired = path(['content', 'blocks', '0', 'versions', 0];
```

Reply:
This logic doesn't seem to work... maybe I am doing something wrong